backstop/scripts/ready.js

```diff
@@ -113,6 +113,33 @@ module.exports = async function (page, scenario, vp) {
     await page.waitForSelector('.cbFormErrorMarker', {visible: true})
   }
 
+  let leafletMapInitialized = await page.evaluate(async function () {
+    let initialized = undefined;
+    var containers = document.querySelectorAll(".js-leaflet-map");
+    if (containers.length) {
+      containers.forEach(function (e) {
+        var container = L.DomUtil.get(e);
+        if (container != null) {
+          if (container._leaflet_id == null) {
+            initialized = false;
```

Comment:
```suggestion
            initialized = false;
```

I'm a little confused by the logic here - by chaining these conditions `containers.length` && `container != null` && `container._leaflet_id == null` means we are only setting `initialized` to false when all three conditions are met, and if any is unmet we are setting `initialized` to true?

Reply:
If the map is not initialized   container._leaflet_id returns null, that's the condition we need to establish a wait for the map to become visible